dataprofiler/profilers/profile_builder.py

```diff
@@ -1519,7 +1520,10 @@ def __init__(
         # Structured specific properties
         self.row_has_null_count = 0
         self.row_is_null_count = 0
-        self.hashed_row_dict: dict = dict()
+        if options.hll_row_hashing:
+            self.hashed_row_dict: HyperLogLog = HyperLogLog(10, sparse=False)
+        else:
+            self.hashed_row_dict: dict = dict()
```

Comment:
Also this doesn't seem like the right way to init since it would be two different types, mypy doesn't like it either. Wonder if we would have two different attributes 🤔

Reply:
might need to do an `|` for the var